etc/config/.env.example

```diff
@@ -31,6 +31,9 @@ BROWSER=chrome
 #FW_BP=
 #TESTS_MODULE_PATH=
 
+#*** Uncomment this property to change the default timezone MFTF will use for the generateDate action ***#
+#DEFAULT_TIMEZONE=America/Los_Angeles
```

Comment:
This is a valid point, but I'd still prefer to keep it generic in case we use this for more actions in the future.
I think that confusion over what this .env does can be solved by descriptive devdocs can be best solved via documentation..